src/components/tables/EmployeeTable.tsx

```diff
@@ -118,9 +112,7 @@ const EmployeeTable: React.FC<EmployeeTableProps> = ({
 									salary={row.salary.toString()}
```

Comment:
Change this, otherwise typescript complains
```suggestion
									salary={row.salary ? row.salary.toString() : ""}
```